pkg/bpmn/persistence_service.go

```diff
@@ -11,15 +13,15 @@ type BpmnEnginePersistenceService interface {
 	FindProcessInstanceByKey(processInstanceKey int64) *processInstanceInfo
 	FindProcessInstances(processInstanceKey int64) []*processInstanceInfo
 
-	FindMessageSubscription(originActivityKey int64, processInstance *processInstanceInfo, elementId string, state ...ActivityState) []*MessageSubscription
-	FindTimers(originActivityKey int64, processInstanceKey int64, state ...TimerState) []*Timer
-	FindJobs(elementId string, processInstance *processInstanceInfo, jobKey int64, state ...ActivityState) []*job
+	FindMessageSubscription(originActivityKey *int64, processInstance *processInstanceInfo, elementId *string, state ...ActivityState) []*MessageSubscription
```

Comment:
Do we need this interface? How many persistence services do we want to have?
We already have https://github.com/pbinitiative/zenbpm/blob/9769cc3dba3e527167430c7e7d6c9684776a8de2/pkg/storage/storage.go#L8 which I think is what we want to use instead of this one